과제2/style2.css

```
@@ -0,0 +1,201 @@
+* {
+  margin: 0;
+  padding: 0;
+  text-align: center;
+}
```

```suggestion
body * {
  margin: 0;
  padding: 0;
  text-align: center;
}
```
* 선택자는 html 모든 요소를 선택하기 때문에 예상치 못 한 에러를 방지하기 위해 body * {} 으로 해주는 것이 더 좋습니다.